make-combinations/io.json

```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "input": [["a","b","c"], [2,4]],
+    "output": true
```

Comment:
The output is wrong here according to the problem description

It should not return true or false, this should return [("a", 1), ("b", 1)....

Reply:
Yes sir, Fixed